airgun/entities/subscription.py

```diff
@@ -55,9 +50,7 @@ def add_manifest(self, manifest_file):
         """
         view = self.navigate_to(self, 'Manage Manifest')
         view.wait_animation_end()
-        view.fill({
-            'manifest.manifest_file': manifest_file,
-        })
+        view.manifest.manifest_file.fill(manifest_file)
```

Comment:
This related to the fact that when filling this field the dialog form disappear and manifest loading begin, this is not how view filing method works, if we let it as the fill method try to access other widgets in the view that lead to exception, to resolve this behavior we two ways,   
1. Do it as I realized here 
2. Inherit fill view method and verify if manifest file is in values, if yes fill it and exit ignoring all other fields otherwise continue with base method.

I chosen the simplest way, as we already know that only that field should be involved and not inherit fill method  :)